lms/templates/goal_reminder_banner.html

```diff
@@ -1,59 +1,18 @@
-{% load i18n %}
 {# email client support for style sheets is pretty spotty, so we have to inline all of these styles #}
-{# we're using important below to override inline styles and my understanding is for email clients where media queries do not work, they'll simply see the desktop css on their phone #}
 <style>
-.goals-two-color:first-line { color: white !important;}
-.goals-two-color {
-    font-size: 2.686rem;
-    line-height: 3rem;
-    margin: 4.5rem 0 3.44rem 1.0625rem;
-    padding-left: 1.25rem;
-    width: 18.75rem;
-    top: 100px;
-}
-@media only screen and (max-width: 575px) {
-  .goals-two-color {
-    font-size: 2.2rem;
-    line-height: 2rem;
-    width: 15rem;
-  }
-}
 @media only screen and (max-width: 456px) {
-  .goals-two-color {
-    font-size: 1.8rem;
-    line-height: 1.8rem;
-    width: 12.5rem;
-    margin: 2.25rem 0;
-    top: 7.5rem;
-  }
-}
-@media only screen and (max-width: 375px) {
-  .goals-two-color {
-    font-size: 1.35rem;
-    line-height: 1.65rem;
-    top: 6.875rem;
-  }
-}
-@media only screen and (max-width: 325px) {
-  .goals-two-color {
-    top: 6.25rem;
+  .goals-banner-div {
+    background-image: url('{{image_url}}goalreminder-mobile-hero.png');
   }
```

Comment:
If this is being applied to the class, wouldn't that make the background-image always the mobile? When does the desktop take precedence?

Reply:
well it is inside a media query with a max-width.
although now that i think about it that wouldn't override the inline style that is being used for desktop so i'll add an important